src/main/java/mate/academy/spring/dto/request/CinemaHallRequestDto.java

```diff
@@ -1,7 +1,15 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import org.hibernate.validator.constraints.Length;
+
 public class CinemaHallRequestDto {
+    @NotNull
+    @Size(min = 20, max = 80)
     private int capacity;
```

Comment:
It's a primitive datatype and it can't be Null
```suggestion
public class CinemaHallRequestDto {
    @Size(min = 20, max = 80)
    private int capacity;
```